src/app/games/quixo/QuixoTutorial.ts

```diff
@@ -1,10 +1,10 @@
 import { QuixoState } from 'src/app/games/quixo/QuixoState';
 import { QuixoMove } from 'src/app/games/quixo/QuixoMove';
 import { Orthogonal } from 'src/app/jscaip/Direction';
-import { Player } from 'src/app/jscaip/Player';
+import { Player, PlayerOrNone } from 'src/app/jscaip/Player';
 import { TutorialStep } from '../../components/wrapper-components/tutorial-game-wrapper/TutorialStep';
 
-const _: Player = Player.NONE;
+const _: PlayerOrNone = PlayerOrNone.NONE;
 const O: Player = Player.ZERO;
 const X: Player = Player.ONE;
```

Comment:
Synthaxiquement légale mais bêêêêêk :D

Reply:
prefer X, O, _ to be 3 times PayerOrNone :p

Reply:
syntaxiquement*, la prochaine fois que tu fais l'erreur on fait quoi ? :D

Reply:
Je suis privé d'épisode de Michel Dumas :D ?